controllers/controller.js

```diff
@@ -80,4 +81,12 @@ exports.deleteCommentById = (req, res, next) => {
       res.status(204).send();
     })
     .catch(next);
-};
\ No newline at end of file
+};
+
+exports.getUsers = (req, res, next) => {
+  selectUsers()
+    .then((users) => {
+      res.status(200).send({ users });
+    })
+    .catch(next);
+};
```

Comment:
Brill! Good model function name and sending the data back on a key of `users` is perfect.